xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/xwiki.js

```diff
@@ -1383,6 +1383,7 @@ if (htmlElement.readAttribute('data-xwiki-reference') != null) {
   XWiki.Document.currentDocumentReference = XWiki.currentDocument.getDocumentReference();
 }
 XWiki.Document.URLTemplate = "$xwiki.getURL('__space__.__page__', '__action__')";
+XWiki.Document.ViewURLTemplate = "$xwiki.getURL('__space__.__page__', 'view')";
```

Comment:
> jumpToPage.js has recently been refactored

I'm not sure what you mean. The only relevant commit on ``jumpToPage.js`` this year is https://github.com/xwiki/xwiki-platform/commit/3bfa9cfcc8b0fafe2cd8cf2326ed5b848f8c963f .